README.md

```diff
@@ -59,6 +60,45 @@ Import sonatype-visualstudio-settings.xml using Tools -> Import and Export Setti
 
 Import sonatype-visualstudio-resharper-settings.xml using ReSharper -> Options... -> Manage... -> Import and Export -> Import from file...
 
+### VSCode
+
+VSCode uses Eclipse formatter settings. To import them clone this repository locally and use path to it in the settings.
+
+- Install Java Extension Pack
+- Configure Java formatter to use Eclipse settings. You need to specify the name of the profile as well (Sonatype).
+- Disable indentation guessing and set it to be forced to 2.
+- Change import order and star thresholds
+- Disable [online services](https://code.visualstudio.com/docs/supporting/FAQ#_how-to-disable-telemetry-reporting) such as telemetry and natural language search.
+
+All settings required as a JSON snippet (make sure to fix the local path):

```

Comment:
How do you import the JSON settings into VS Code?

Reply:
Code -> Preferences -> then press the paper icon with the curvey arrow at the top, to put it in JSON mode? Then whenre in the JSON do you add this?

Reply:
Yes, it is all top-level.

Here is the documentation and the default settings file at the bottom:
https://code.visualstudio.com/docs/getstarted/settings

I do not think this worth mentioning in the setup steps though. Since after changing any setting in the UI user will get a non-empty json file from which it is clear where to add them.

Reply:
IntelliJ and Eclipse look like they link to help pages for settings. The link you found above seems like it could be a good fit. Maybe say "look at this link to see how to import the settings" or something like that?